vacs-map-app/src/utils/colors.js

```diff
@@ -17,8 +17,20 @@ export const stopLightScheme = {
   default: '#7E899C'
 }
 
+// need at least 7 for now
+export const ordinalScheme = [
+  '#F7B300',
```

Comment:
@mwbernard could we pin the first color (golden/yellow) to always be the reference crop? this would match the UI nicely